### PR TITLE
Document `observe_value` and `observe_lr` trigger interval

### PR DESCRIPTION
**Pull request by niboshi**
They're triggered at non-default interval of `(1, 'epoch')`, but it's not documented.

### CODE REVIEW

File: chainer/training/extensions/value_observation.py [hunk 1]
```diff
@@ -10,6 +10,11 @@ def observe_value(observation_key, target_func):
             It must take one argument: :class:~chainer.training.Trainer object.
     Returns:
         The extension function.
+
+    This extension is triggered every 1 epoch by default.
```
Comment (Crissman) [inline chainer/training/extensions/value_observation.py]:
```suggestion
    This extension is triggered each epoch by default.
```

File: chainer/training/extensions/value_observation.py [hunk 2]
```diff
@@ -10,6 +10,11 @@ def observe_value(observation_key, target_func):
...
+    To change this, specify ``trigger`` argument to
```
Comment (Crissman) [inline chainer/training/extensions/value_observation.py]:
```suggestion
    To change this, use the ``trigger`` argument with the
```

File: chainer/training/extensions/value_observation.py [hunk 3]
```diff
@@ -28,6 +33,11 @@ def observe_lr(optimizer_name='main', observation_key='lr'):
 
     Returns:
         The extension function.
+
+    This extension is triggered every 1 epoch by default.
```
Comment (Crissman) [inline chainer/training/extensions/value_observation.py]:
```suggestion
    This extension is triggered each epoch by default.
```

File: chainer/training/extensions/value_observation.py [hunk 4]
```diff
@@ -28,6 +33,11 @@ def observe_lr(optimizer_name='main', observation_key='lr'):
...
+    To change this, specify ``trigger`` argument to
```
Comment (Crissman) [inline chainer/training/extensions/value_observation.py]:
```suggestion
    To change this, use the ``trigger`` argument with the
```

**Comment by niboshi** [comment]
@Crissman Thank you for the review. It has been applied.

**Comment by Crissman** [comment]
Jenkins, test this please.

**Comment by Crissman** [comment]
Jenkins build didn't run. Retrying. Jenkins, test this please.

**Comment by Crissman** [comment]
Jenkins failure appears unrelated.

**Pull request merged**